yarn.lock

```diff
@@ -1,215 +0,0 @@
-# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
@xavier506 why was this file deleted?

Reply:
I'm guessing it's because the front-end was moved to `services/frontend` but just want to make sure we don't really need this anymore.

Reply:
It got deleted because all package files are under the services directory , it is an orphaned file.